adafruit_pybadger.py

```diff
@@ -227,9 +232,16 @@ def show_business_card(self, image_name=None, dwell=20):
             on_disk_bitmap = displayio.OnDiskBitmap(file_name)
             face_image = displayio.TileGrid(on_disk_bitmap, pixel_shader=displayio.ColorConverter())
             business_card_splash.append(face_image)
-            # Wait for the image to load.
             self.display.wait_for_frame()
-            time.sleep(dwell)
+        email_group = displayio.Group(scale=email_scale)
+        email_label = Label(email_font, text=email_string)
+        (_, _, width, height) = email_label.bounding_box
+        email_label.x = ((self.display.width // (2 * email_scale)) - width // 2)
+        email_label.y = int(height // (0.13 * email_scale))
+        email_label.color = 0xFFFFFF
+        email_group.append(email_label)
+        business_card_splash.append(email_group)
+        time.sleep(dwell)
```

Comment:
I'm not entirely sure of the point of the automatically sleep for 20 seconds is. If this is for the purposes of the simepletest, would it be more useful to put it in the simpletest itself?

Reply:
I realised earlier that I need to make the code non-blocking, but didn't have the time to do so yet. Please do not merge until I have done that - I'll get to it this evening.